helm/hpcc/templates/_helpers.tpl

```diff
@@ -1337,6 +1337,67 @@ spec:
 {{- end }}
 {{- end }}
 
+{{/*
+Use cert-manager to create a public certificate and private key for use as
+remote client certificates.
+Adding the following to ESP, Roxie, or dafilesrv
+  remoteClients:
+  - name: myRemoteClient
+Will generate secrets containing certificates that can be deployed to the remote client.
+If ESP or Roxie is configured to require client certificates then these certificates
+can be used to allow those remote clietns to connect.  Unique certificates should be created
+for each client so we can start to create certificate based access control lists in the near future.
+*/}}
+{{- define "hpcc.addClientCertificate" }}
+{{- if (.root.Values.certificates | default dict).enabled -}}
+{{- $externalCert := or (and (hasKey . "external") .external) (ne (include "hpcc.isVisibilityPublic" .) "") -}}
```

Comment:
indent: minor comment, would should the structure more clearly if the ifs were indented - line 1365 is for some reason.